apps/volontulo/tests/models/test_offerstatus.py

```diff
@@ -0,0 +1,115 @@
+from apps.volontulo.models import Organization
+
+
+class TestOfferStatusModel(TestCase):
+    u"""Tests for OfferStatus model."""
+
+    @classmethod
+    def setUpTestData(cls):
+        u"""Fixtures for OfferStatus model unittests."""
+        from datetime import datetime
+        from datetime import timedelta
+
+        current_datetime = '2015-11-01 11:01:11'
+        past = datetime.strptime(
+            current_datetime,
+            "%Y-%m-%d %H:%M:%S"
+        ) - timedelta(days=5)
+        future = datetime.strptime(
+            current_datetime,
+            "%Y-%m-%d %H:%M:%S"
+        ) + timedelta(days=5)
+
+        # closed offer
+        Offer.objects.create(
+            organization=Organization.objects.create(
+                name=u'Some organization',
+                address=u'',
+                description=u''
+            ),
+            started_at=str(past - timedelta(days=2)),
+            finished_at=str(past),
+            description=u'',
+            requirements=u'',
+            time_commitment='12.12.2015',
+            benefits=u'',
+            location=u'',
+            title=u'',
+            time_period=u'',
+        )
+
+        # future offer
+        Offer.objects.create(
+            organization=Organization.objects.create(
+                name=u'Some organization',
+                address=u'',
+                description=u''
+            ),
+            started_at=str(future),
+            finished_at=str(future + timedelta(days=2)),
+            description=u'',
+            requirements=u'',
+            time_commitment='12.12.2015',
+            benefits=u'',
+            location=u'',
+            title=u'',
+            time_period=u'',
+        )
+
+        # ongoing offer (start + end date)
+        Offer.objects.create(
+            organization=Organization.objects.create(
+                name=u'Some organization',
+                address=u'',
+                description=u''
+            ),
+            started_at=str(past),
+            finished_at=str(future),
+            description=u'',
+            requirements=u'',
+            time_commitment='12.12.2015',
+            benefits=u'',
+            location=u'',
+            title=u'',
+            time_period=u'',
+        )
+
+        # ongoing offer (start date, no ending before now)
+        Offer.objects.create(
+            organization=Organization.objects.create(
+                name=u'Some organization',
+                address=u'',
+                description=u''
+            ),
+            started_at=str(past),
+            finished_at=None,
+            description=u'',
+            requirements=u'',
+            time_commitment='12.12.2015',
+            benefits=u'',
+            location=u'',
+            title=u'',
+            time_period=u'',
+        )
+
+    def test__determine_action_status(self):
+        u"""Verify action status."""
+        finished_offer = Offer.objects.get(id=1)
```

Comment:
Don't rely on the IDs. For example, you can assign offers in the setUpTestData to cls.offer1, etc, and here use `Offer.objects.get(id=self.offer1.id). Or search by something else than ID.